scripts/ddr_summary_figures.R

```diff
@@ -17,18 +17,19 @@ library(pheatmap)
 library(ggplot2)
 source(file.path("scripts", "util", "pancancer_util.R"))
 
-results <- parse_summary(file.path("classifiers", "TP53",
-                                   "classifier_summary.txt"))
+results_folder <- file.path("classifiers", "TP53")
+results <- parse_summary(file.path(results_folder, "classifier_summary.txt"))
 
 # 1) Heatmap of the distribution of aberrant events across tumors
-heat_df <- readr::read_tsv(file.path("tables",
-                                     "TP53_mutations_heatmap_data.csv"))
-prop_matrix <- as.matrix(heat_df[,c(3, 2)])
+heat_file <- file.path(results_folder, "summary_counts.csv")
+heat_df <- readr::read_csv(heat_file)
+
+prop_matrix <- as.matrix(heat_df[, c(6, 5)])
```

Comment:
Does `heat_df` have column names? It's usually better to access columns by names than indices.